platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceBean.java

```diff
@@ -421,4 +467,20 @@ protected BundleContext getContext() {
   void setServiceTracker(ServiceTracker serviceTracker) {
     this.serviceTracker = (ServiceTracker<Object, Object>) serviceTracker;
   }
+
+  /**
+   * Exception to throw to ensure Jolokia registers an error. Also clears the stacktrace so details
+   * do not leak to the UI.
+   */
+  private class ApplicationServiceBeanException extends RuntimeException {
```

Comment:
❗️Instead of creating our own Exception that strips the stack trace, we should consider using the `includeStackTrace` (or `allowErrorDetails`) options in the Jolokia servlet context configuration, or the `org.jolokia.includeStackTrace` in the OSGi bundle configuration, to exclude all stack traces from responses. Since I think we never want those going back out, that would be a much better option imho.

Reply:
I looked into this, the `includeStackTrace` is a request parameter and doesn't prevent the backend end from exposing stacktraces. The `allowErrorDetails` configuration was introduced in 1.3.3, we're currently on 1.2.3. The latest version is 1.5.5. I think a dependency bump that large is a bit beyond the scope of this pr.